eclipse-collections-code-generator/src/main/resources/impl/map/mutable/objectPrimitiveHashMap.stg

```diff
@@ -1129,6 +1131,23 @@ public class Object<name>HashMap\<K> implements MutableObject<name>Map\<K>, Exte
         return new KeyValuesView();
     }
 
+    <if(!primitive.booleanPrimitive)>
+    @Override
+    public Mutable<name>ObjectMap\<K> flipUniqueValues()
+    {
+        Mutable<name>ObjectMap\<K> result = <name>ObjectMaps.mutable.empty();
+        this.forEachKeyValue((key, value) ->
+        {
+            K oldKey = result.put(value, key);
+            if (oldKey != null)
+            {
+                throw new IllegalStateException("Duplicate value: " + value + " found at key: " + oldKey + " and key: " + key);
+            }
+        });
+         return result;
```

Comment:
CheckStyle violation on one of these lines since they don't line up.

Reply:
Fixed!